### PR TITLE
[systemtest] Fix tests failing in nightlies and tests in acceptance

### PR DESCRIPTION
**Pull request by im-konge**
Signed-off-by: Lukas Kral <lukywill16@gmail.com>

### Type of change

- Bugfix

### Description

In this PR I'm gonna fix some tests failing in nightlies and acceptance profile:

`MetricsST.testKafkaTopicPartitions` - after #3468 the CC was added to the suite, so new topics are created with another rep.factor and partitions, so I change it to correct value -> from `291` partitions to `420` (after sum)
`MetricsST.testTopicOperatorMetrics` - as I mentioned above, after #3468 new topics were created and weren't added to `getExpectedTopics`
`UserST.testTlsUserWithQuotas` - here was problem after #3490 (I guess) with connecting to zookeeper -> so I change to getting all the info from Zookeeper pod
`UserST.testScramUserWithQuotas` - same as above

All CC tests: I added `@Disabled` for skipping these tests in nightly runs until the fix for Kafka `2.6.0` will be merged and new version of CC will be released.

### Checklist

- [x] Write tests
- [x] Make sure all tests pass

### CODE REVIEW

**Comment by im-konge** [comment]
@strimzi-ci run tests test_only

**Comment by im-konge** [comment]
@strimzi-ci run tests test_only

File: systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlApiST.java [hunk 1]
```diff
@@ -30,6 +31,7 @@
 @Tag(REGRESSION)
 @Tag(CRUISE_CONTROL)
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@Disabled
```
Comment (see-quick) [inline systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlApiST.java:34]: Here write some commend why do you Disabled these tests and same in ConfigurationST...Thanks

Comment (im-konge) [reply systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlApiST.java:36]: I added to all 3 suites :) thanks for this reminder

**Comment by im-konge** [comment]
@strimzi-ci run tests test_only

**Pull request merged**